### ISSUE TITLE
Bug in Offline Support

### ISSUE THREAD

**Issue by p-sims** [johannes-schliephake/nextcloud-passwords-ios#286]
### iOS App Version

2.7.1

### iOS Version

15.8.1

### Device Model

iPhone SE 2017

### Nextcloud Server Version

27.1.7

### Server App Version

2023.12.32

### Peculiarities of Your Setup

End 2 End encryption enabled.

### Observed Behavior

When phone is connected to internet everything works as expected. When app is opened phone asks for phone passcode, it then asks for E2E password, and then in should you your password decrypted.

However if you do the same steps in AirPlane Mode, you end up with this:
![2024-05-30_12-17_PM4079](https://github.com/johannes-schliephake/nextcloud-passwords-ios/assets/64636731/813146cb-1379-40d1-8c77-f9f6cca8d328)

### Expected Behavior

App should work the same both online and offline it should just note that's it's offline so changes won't sync....

### Steps to Reproduce

1. Turn off all internet connections
2. Ensure app is not running (swipe up to close it if it is)
3. Open the app
4. Put in phone passcode
5. Put in E2E Password

### Debugging

- [ ] I did install the [TestFlight](https://testflight.apple.com/join/iuljLJ4u) version to check the log on the app's *Settings* page → *Log*.
- [X] I did check for iOS logs named *Passwords* under *System Settings* → *Privacy & Security* → *Analytics & Improvements* → *Analytics Data*.
- [X] I did check for Nextcloud logs on the admin's *Logging* page.

### Logs

```shell
Not found anything relevent...

I install TestFlight app but needed a code to redeem testflight version of passwords app so draw a blank.
```

**Comment by johannes-schliephake** [comment]
Yeah, i see what you're getting at. The app doesn't work exactly the same when offline. If a password can't be created on the server, the red triangle is shown. Currently there is no mechanism to try again as soon as the device is online again. You have to tap "Edit" on such an entry again and just save it again.

Definitly not the best user experience. I will change that.
